test/GPv2AllowanceManager.test.ts

```diff
@@ -32,10 +32,9 @@ describe("GPv2AllowanceManager", () => {
     });
 
     it("should execute ERC20 transfers", async () => {
-      const NonStandardERC20 = await artifacts.readArtifact("NonStandardERC20");
       const tokens = [
         await waffle.deployMockContract(deployer, IERC20.abi),
-        await waffle.deployMockContract(deployer, NonStandardERC20.abi),
+        await waffle.deployMockContract(deployer, IERC20.abi),
```

Comment:
Why are we no longer checking for non standard ERC20s here? Is it not needed?

Reply:
We have unit tests for all the non-standard ERC20 paths in `GPv2TradeExection.sol` already and felt a little overkill.